providers/common/ciphers/build.info

```diff
@@ -12,8 +12,10 @@ $COMMON=cipher_common.c cipher_common_hw.c block.c \
         cipher_ccm.c cipher_ccm_hw.c \
         cipher_aes_ccm.c cipher_aes_ccm_hw.c \
         cipher_aes_wrp.c \
+        cipher_aes_cbc_hmac_sha.c cipher_aes_cbc_hmac_sha_hw.c \
+        cipher_aes_cbc_hmac_sha256_hw.c cipher_aes_cbc_hmac_sha1_hw.c \
```

Comment:
I can see already here that you need to catch up. This should be providers/implementations/ciphers/build.info

Reply:
I might be doing that now :)